src/SFA.DAS.Forecasting.Web/Views/Forecasting/Balance.cshtml

```diff
@@ -65,6 +65,12 @@
 <!-- CHART -->
 <div class="grid-row">
     <div class="column-full">
+        @if (@Model.ProjectionDate.HasValue)
+        {
+            <div class="panel projection-date">
+                <p>Your projeciton was last updated on @Model.ProjectionDate.Value.ToGdsFormatFull()</p>
+            </div>
+        }
```

Comment:
Might be worth getting a default message for this for if your projection has never been created.